src/components/ReferenceData.vue

```diff
@@ -20,7 +20,9 @@
 					:options="languages"
 					placeholder="Select language"
 					label="language"
-					class="lang-select"/>
+					class="lang-select"
+					:disabled="readOnly"
```

Comment:
where is readOnly?

Reply:
it's a prop I added in base.vue, so everything that extends the base component has it

Reply:
I initially considered calling it disabled that's typically the name of the prop for disabling fields, but since the buttons that don't modify data (e.g. changing tabs) are still functioning, calling the components disabled didn't feel right.